interview-questions.md

```diff
@@ -7,48 +7,50 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is an instance variable in Ruby? How is it different from other variables?
 
-  Your answer:
+  Your answer: It is a snapshot of the object in a class
 
-  Researched answer:
+  Researched answer: The instance variable is an object in a class with a local scope. 
 
 
 
 2. What is a block in Ruby?
 
-  Your answer:
+  Your answer: I assumed it was a block of code in Ruby. 
 
-  Researched answer:
+  Researched answer: A ruby block acts similarly to a method but without a name, which offers liberties that js methods can't offer.
+  " a block is an anonymous piece of code, it can accept input in form of arguments (if it needs any), and it will return a value, but it does not have a name."
 
 
 
 3. Ruby has an implicit return. What does that mean?
 
-  Your answer:
+  Your answer: I do not know.
 
-  Researched answer:
+  Researched answer: implicit return is a behaviour that results in the output of a method returning the last line of code
 
 
 
 4. What is object-oriented programming? How is it different than functional programming?
 
-  Your answer:
+  Your answer: I'm not entirely sure what this means and difference between the two terms but I imagine that object oriented programming is self explanatory, in that the program is centered on objects, whereas functional programming is centered on functions.
 
-  Researched answer:
+  Researched answer: Ruby is an example of object oriented programing as it processes every value as an object. In functional programming, data cannot be stored in objects and it can only be transformed by creating functions. 
 
 
 
 5. What is the difference between a class and an object?
 
-  Your answer:
+  Your answer: class is a blueprint for objects. Objects are passed through classes.
```

Comment:
I would encourage you to go into more depth with these answers. The interviewer would, 9 times out of 10, ask you to elaborate further, which you can avoid having them do (as we all know, that can be annoying for you and for the person interviewing you,  to feel like you need to repeat yourself) by making sure you're covering as much ground as possible with your answer.  

That being said, your answer is not wrong, just a bit short!